Guardfile

```diff
@@ -3,7 +3,7 @@
 
 notification :growl
 
-guard :minitest, spring: true, env: {GUARD: 'true'}, all_env: :GUARD_COVERAGE, all_after_pass: true do
+guard :minitest, spring: true, env: {GUARD: 'true'}, all_env: :GUARD_COVERAGE, all_after_pass: false do
```

Comment:
just too slow? I usually like having all run after changes, but understandable to change this.